image/templates/helm/stackrox-central/templates/_init.tpl.htpl

```diff
@@ -139,6 +139,14 @@
 {{ include "srox.autoSensePodSecurityPolicies" (list $) }}
 [<- end >]
 
+[< if .Operator >]
+{{ $_ := set $env "installMethod" "rhacs-operator" }}
```

Comment:
Not sure, do we avoid the naming `rhacs` here and simply do `operator`? In the future if we were to introduce an operator for stackrox, then this would clash probably.